chapters/data-processing.tex

```diff
@@ -0,0 +1,784 @@
+
+% What is data cleaning
+Implementing the tasks described in this chapter will make data analysis much easier.
+The first section will guide on making your data \textit{tidy},
+so the relationship between rows and columns in a data set is well-defined.
+This includes removing duplicated observations.
+To assert the accuracy of your data, it must then go through quality assurance checks,
+as described in the second section.
+After that, identifying information is typically no longer needed,
+the following section will outline steps to remove direct identifiers.
+Finally, the last section discussed how to explore each variable in your dataset and 
+make sure that it is as well documented and easy to use as possible.
+
+
+\end{fullwidth}
+
+%------------------------------------------------
+
+\subsection{Before you get started}
+
+% Don't mess with the raw data
+The starting point for all tasks described in this chapter is the raw data,
+containing the exact files received, with no changes or additions.
+Be mindful of how and where these files are stored
+as they cannot be re-created and nearly always contain confidential information.
+Raw data files must \textit{never} be edited directly.
+If the raw data contains confidential information,
+any tasks that precede de-identification must be carried on encrypted data.
+ 
+% Why cleaning is necessary
+Raw data files may be received in a variety of formats, 
+and are often not easy to handle using statistical software.
+The data cleaning tasks described in this chapter will prepare the raw data
+for quantitative analysis.
+To make this complex process less error-prone,
+we recommend that no changes be made to the content of data at this point
+(with one exception that will be carefully discussed).
+Correcting outliers, imputing values and creating derived indicators
+are all research decisions, and are therefore considered part of analysis.
+Therefore, the main output of data cleaning, the cleaned dataset,
+contains almost exactly same information as the raw data,
+but in a format that is easier to interact with.  
+
+\section{Identifying the identifier(s)}
+
+% Intro
+The very first step in creating statistical software-friendly data sets
+is understanding the data acquired,
+and translating this understanding into an intuitive format.
+This section discusses what steps may be needed to make sure that each row
+in a dataset represents one observation.
+This may seem trivial, but should not be taken for granted.
+Getting to such a format may be harder than expected,
+and the \textbf{unit of observation}\sidenote{\textbf{Unit of observation:}
+	the unit described by the data. \url{
+		https://dimewiki.worldbank.org/wiki/Unit_of_Observation}}
+ may be ambiguous in many raw datasets.
+But having data in a tidy format is the best way to manipulate it, 
+particularly aggregating and cross-referencing.
+You will need to do manipulate data as soon as it is received,
+so you can assure its quality.
```

Comment:
sentence has an error, doesn't make sense as is but i'm not sure how to correct

Reply:
Is this a way to say what you want to say?
"_To be able to quickly assure the quality of the data, it is important that you do this manipulation immediately as you receive it._"

Reply:
Made an attempt on this in 6c688d8